trl/trainer/dpo_config.py

```diff
@@ -103,7 +103,9 @@ class DPOConfig(TrainingArguments):
         learning_rate (`float`, *optional*, defaults to `1e-6`):
             Initial learning rate for [`AdamW`] optimizer. The default value replaces that of
             [`~transformers.TrainingArguments`].
-        loss_type (`str`, *optional*, defaults to `"sigmoid"`):
+        loss_weights (`dict`, *optional*, defaults to `{"sigmoid": 1.0}`):
+            Use to weight a combination of losses.
```

Comment:
Maybe add that 
- the keys in `loss_weights` must be in `loss_type`
- by default (if not specified in the dict), the weight for a loss in `loss_type` is `1.0`